libs/platform/experience/src/routes.ts

```diff
@@ -15,27 +15,31 @@ export const defaultExperienceRoutes: RouteConfig[] = [
       route="/product/:sku"
     ></oryx-composition>`,
     name: 'Product test',
+    type: RouteLinkType.Product,
```

Comment:
Can we drop the name on these routes? Where do we use it?

Reply:
Stan I also discussed to drop the `<oryx-composition>` by default. Have you considered this?